apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.drop", repo
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+Mix.Task.run "run", ~w(apps/yggdrasil/priv/repo/seeds.exs)
```

Comment:
Could you build the path up? Maybe [`__DIR__`](http://elixir-lang.org/docs/stable/elixir/Kernel.SpecialForms.html#__DIR__/0)? Something like:

```elixir
Path.join([__DIR__, "..", "..", "apps", "yggdrasil", "priv", "repo", "seed.ex"])
```

Or maybe using the [`Application.app_dir/2`](http://elixir-lang.org/docs/stable/elixir/Application.html#app_dir/2)?

```elixir
Application.app_dir :Yggdrasil, Path.join(["priv", "repo", "seed.ex"])
```

Also I wanted to point out that I used the `Path.join` because that should eliminate issues with different path separators on Windows vs Unix. But I don't know if Elixir makes you have to do this or not. Nodejs requires you to build the path in a platform agnostic way otherwise BOOM! when you go to run on a different OS.